tools/cldr-apps/src/main/java/org/unicode/cldr/web/Reports.java

```diff
@@ -0,0 +1,56 @@
+package org.unicode.cldr.web;
+
+import java.util.EnumSet;
+import java.util.Set;
+import java.util.TreeSet;
+
+import org.unicode.cldr.util.VoterReportStatus.ReportId;
+
+/**
+ * Utilities for managing report status in settings
+ */
+public class Reports {
+    public static String getSettingsName(ReportId r) {
```

Comment:
could these be private? so far, only called locally

Reply:
I'll make them private for now.